kani-compiler/src/main.rs

```diff
@@ -105,6 +105,14 @@ fn main() -> Result<(), &'static str> {
     // Generate rustc args.
     let rustc_args = generate_rustc_args(&matches);
 
+    if matches.is_present(parser::ENABLE_STUBBING) {
+        assert!(
+            matches.is_present(parser::HARNESS),
+            "Flag --enable-stubbing requires --harness flag"
+        );
+        eprintln!("warning: Kani currently does not perform any stubbing.");
```

Comment:
This warning is checked in `tests/cargo-ui/stubbing-flag/expected` and `tests/ui/stubbing-flag/expected`.

Reply:
Sorry, I meant a test case for the assertion above:
```
        assert!(
            matches.is_present(parser::HARNESS),
            "Flag --enable-stubbing requires --harness flag");
```
In fact, isn't it dead code? I'd assume the parser does already detect this case (because of `requires("harness")` in the `structopt` declaration.

Reply:
It's not dead code, because the `structopt` declaration is for the `kani-driver` args, not the `kani-compiler` args (which are being checked here). I wanted an internal assertion here to make sure that `kani-driver` is actually passing both args to `kani-compiler`.

However, it looks like I can actually specify the `kani-compiler` argument as requiring `--harness`, which would be cleaner and means that I can get rid of that assertion.

Would we still want a test for that case?

Reply:
Actually, it's easy to add a test - I'll do that.

Reply:
Yes, I think it's cleaner to declare the requirement for the `kani-compiler` argument. Wouldn't need a test in that case.